app/models/miq_widget_set.rb

```diff
@@ -69,6 +69,15 @@ def self.sync_from_file(filename)
     end
   end
 
+  def self.copy_dashboard(original_dashboard, new_name, new_tab_title, assign_to_group_id = nil)
```

Comment:
Discussed with @yrudman and we decided to rename these to:
```
 def self.copy_dashboard(source_widget_set, destination_name, destination_description, assign_to_group_id = nil)
```

Reply:
done